docs/pages/eas-build-custom-builds/custom-build-functions.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Custom JavaScript/TypeScript functions
+description: Learn how to create and use custom JavaScript/TypeScript functions in your EAS Build custom builds configuration.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+Custom functions are a great way to extend the functionality of EAS Build custom builds. You can use them to write reusable pieces of custom build config in a very powerful and convenient way.
+
+<Step label="1">
+
+## Creating custom JavaScript/TypeScript functions
```

Comment:
```suggestion
## Create custom JavaScript/TypeScript functions
```

Let's use short form verbs instead of gerunds for section titles for consistency reasons.